alerting/src/main/kotlin/org/opensearch/alerting/service/DeleteMonitorService.kt

```diff
@@ -24,9 +24,9 @@ import org.opensearch.action.support.WriteRequest.RefreshPolicy
 import org.opensearch.action.support.master.AcknowledgedResponse
 import org.opensearch.alerting.MonitorMetadataService
 import org.opensearch.alerting.opensearchapi.suspendUntil
-import org.opensearch.alerting.transport.TransportDeleteWorkflowAction.Companion.WORKFLOW_DELEGATE_PATH
-import org.opensearch.alerting.transport.TransportDeleteWorkflowAction.Companion.WORKFLOW_MONITOR_PATH
 import org.opensearch.alerting.util.AlertingException
+import org.opensearch.alerting.util.ScheduledJobUtils.Companion.WORKFLOW_DELEGATE_PATH
+import org.opensearch.alerting.util.ScheduledJobUtils.Companion.WORKFLOW_MONITOR_PATH
```

Comment:
can we move this under `util` package instead as the `service` package contains only 1 file `DeleteMonitorService`.

Reply:
typically i favor this package formatting of rest, transport, service layer.

Intention is to migrate more code to these kind of classes. This enables writing functional tests for service classes. 

Keeping as is. We can discuss more if you feel there is a better way